package.json

```diff
@@ -61,7 +61,7 @@
     "serverless-security": "node scripts/kibana --dev --serverless=security",
     "spec_to_console": "node scripts/spec_to_console",
     "start": "node scripts/kibana --dev",
-    "storybook": "node scripts/storybook",
+    "storybook": "node --openssl-legacy-provider scripts/storybook",
```

Comment:
Canvas has a static runtime that may need to be updated similarly :
`node x-pack/plugins/canvas/scripts/shareable_runtime.js`

https://github.com/elastic/kibana/pull/149165 for the prior fix.